src/web/src/sse.cc

```diff
@@ -281,11 +281,13 @@ std::string Sse::changeFtyProtoAsset2Json(fty_proto_t *asset)
     json += "data:{\"topic\":\"asset/" + nameElement + "\",\"payload\":{}}\n\n";
   }
   else if (streq(fty_proto_operation(asset), FTY_PROTO_ASSET_OP_UPDATE)
+          || streq(fty_proto_operation(asset), FTY_PROTO_ASSET_OP_INVENTORY)
           || streq(fty_proto_operation(asset), FTY_PROTO_ASSET_OP_CREATE))
   {
-    log_debug("Sse get an update or create message");
+    log_debug("Sse get an update, create or inventory message");
```

Comment:
"got"? "SSE"?

Reply:
yeah, made the same comment to myself... but was too lazy :D